app/models/carto/profile.rb

```diff
@@ -0,0 +1,15 @@
+# encoding: UTF-8
+
+module Carto
+  class Profile < ActiveRecord::Base
+
+    validates :name, presence: true
+
+    has_many :profiles_users, :dependent => :destroy
```

Comment:
Rocket operators are mostly deprecated. You can use `Rubocop` to lint the file with our config and tell you about these things :)

Should read:

`has_many :profiles_users, dependent: :destroy`